sdk/storage/azure-storage-common/src/crypt.cpp

```diff
@@ -1,15 +1,15 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // SPDX-License-Identifier: MIT
 
-#include <azure/core/platform.hpp>
 #include "azure/storage/common/crypt.hpp"
+#include <azure/core/platform.hpp>
 
-#ifdef AZ_PLATFORM_WINDOWS
+#if defined(AZ_PLATFORM_WINDOWS)
 #ifndef NOMINMAX
 #define NOMINMAX
 #endif
-#include <windows.h>
 #include <bcrypt.h>
+#include <windows.h>
```

Comment:
don't reorder these two headers. it will cause build errors

Reply:
In that case, please add a comment and leave a new line between the two include statements. Otherwise, every time someone opens in VS and re-formats it, the header order will change and break the build.